starter_code/app.js

```diff
@@ -1,18 +1,51 @@
+'use strict';
 
 const express = require('express');
-const hbs     = require('hbs');
-const app     = express();
-const path    = require('path');
+const hbs = require('hbs');
+const app = express();
+const path = require('path');
```

Comment:
try to organise it like 1st requirements, empty line, then app = express();, blah..